### PR TITLE
Add additional CRUD tests for Chime

### PR DESCRIPTION
**Pull request by davidcui1225**
Signed-off-by: David Cui <davidcui@amazon.com>

Add additional tests to Chime CRUD tests to test more use cases/edge cases.
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by joshuali925 — DISMISSED** [review]

File: notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt [hunk 1]
```diff
@@ -265,4 +265,128 @@ class ChimeNotificationConfigCrudIT : PluginRestTestCase() {
...
+        {
+            "config":{
+                "name":"${referenceObject.name}",
+                "description":"${referenceObject.description}",
+                "config_type":"chime",
+                "feature_list":[
+                    "${referenceObject.features.elementAt(0)}"                
+                ],
+                "is_enabled":${referenceObject.isEnabled},
+                "chime":{"url":"http"}
```
Comment (zhongnansu) [inline notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt:291]: shouldn't this url field come from referenceObject?

File: notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt [hunk 2]
```diff
@@ -265,4 +265,128 @@ class ChimeNotificationConfigCrudIT : PluginRestTestCase() {
...
+        }
+        """.trimIndent()
+        executeRequest(
+            RestRequest.Method.POST.name,
+            "$PLUGIN_BASE_URI/configs",
+            createRequestJsonString,
+            RestStatus.INTERNAL_SERVER_ERROR.status
+        )
+    }
+    fun `test BAD delete request on non-existent config ID`() {
```
Comment (zhongnansu) [inline notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt:302]: add 1 line spacing between functions?

File: notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt (hunk 1 above)
Comment (davidcui1225) [reply notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt:291]: No, if the referenceObject is provided an invalid URL the test case will throw failure.

**Review by zhongnansu — DISMISSED** [review]

File: notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt (hunk 2 above)
Comment (davidcui1225) [reply notifications/notifications/src/test/kotlin/org/opensearch/integtest/config/ChimeNotificationConfigCrudIT.kt:302]: Will add

**Review by joshuali925 — APPROVED** [review]
